.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,76 @@
+name: Bug report
+description: Create a report to help us improve
+body:
+  - type: markdown
+    attributes:
+      value: >
+        DuckDB-Wasm is part of the wider DuckDB ecosystem split in different repos, please make sure you're raising your issue in the correct one:
+         * [DuckDB core](https://github.com/duckdb/duckdb/issues/new)
+         * [Our docs/website](https://github.com/duckdb/duckdb-web/issues/new)
+
+        If neither seems to be applicable, you are in the right place!
+
+  - type: textarea
+    attributes:
+      label: What happens?
+      description: A short, clear and concise description of what the bug is.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: To Reproduce
+      description: Steps to reproduce the behavior. Bonus points if those are only SQL queries.
```

Comment:
Should we a ask for a link to a reproduction (we could provide an observable template).

Reply:
For shell.duckdb.org at some point there would be a URL-encoded way, that we should definitively add. Observable probably a link is sufficient, other deployments depends